sdk/openai/azure-openai/samples/README.md

```diff
@@ -0,0 +1,70 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure-openai
+urlFragment: openai-samples
+---
+
+# Samples for OpenAI client library for Python
+
+These code samples show common scenario operations calling to Azure OpenAI.
+
+You can authenticate your client with an API key or through Azure Active Directory with a token credential from [azure-identity][azure_identity].
+
+These sample programs show common scenarios for using Azure OpenAI offerings.
+
+|**File Name**|**Description**|
+|----------------|-------------|
+|[chat_completions_quickstart.py][chat_completions_quickstart]|Use Chat Completions with Azure OpenAI|
+|[chat_completions_oyd.py][chat_completions_oyd]|Use Chat Completions with Azure OpenAI on your data|
+
+## Prerequisites
+
+* Python 3.8 or later is required to use this package
```

Comment:
The Azure SDK for Python requires 3.8 or greater, but the openai library specifies 3.7 or later